pkg/datapath/linux/node.go

```diff
@@ -1401,8 +1399,10 @@ func (n *linuxNodeHandler) NodeCleanNeighborsLink(l netlink.Link, migrateOnly bo
 		defer n.neighLock.Unlock()
 	}
 
-	var neighSucceeded, neighErrored int
-	var which string
+	var (
+		neighSucceeded, neighErrored int
+		which                        string
+	)
```

Comment:
I'd avoid making changes like this and the one on line 1287. They are style preferences for which the Cilium project doesn't have rules AFAIK. They don't fix anything but can cause conflicts on backports & co. IMO, the drawbacks outweigh the benefits.

Reply:
I agree besides DRY and readability no value add here. I'll revert it